### PR TITLE
[docs] Create better navigation on index

### PR DESCRIPTION
**Pull request by stevhliu**
This PR updates the main docs landing page with better navigation around the docs (some of the buttons link to the `Overview` pages which aren't merged yet), and removes links to Colab notebooks in the table of supported pipelines.

### CODE REVIEW

File: docs/source/en/index.mdx [hunk 1]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
     <br>
 </p>
 
-# 🧨 Diffusers
+# 🤗 Diffusers
 
-🤗 Diffusers provides pretrained vision and audio diffusion models, and serves as a modular toolbox for inference and training.
+🤗 Diffusers is the go-to library for state-of-the-art pretrained diffusion models for generating images, audio, and even 3D structures of molecules. Whether you're looking for a simple inference solution or training your own diffusion models, 🤗 Diffusers is a modular toolbox that supports both. Our library is designed with a focus on [usability over performance](conceptual/philosophy#usability-over-performance), [simple over easy](conceptual/philosophy#simple-over-easy), and [customizability over abstractions](conceptual/philosophy#tweakable-contributorfriendly-over-abstraction).
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:21]: Damn, this reads so good!

File: docs/source/en/index.mdx [hunk 2]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
 
-More precisely, 🤗 Diffusers offers:
+🤗 Diffusers offers three core components:
 
-- State-of-the-art diffusion pipelines that can be run in inference with just a couple of lines of code (see [**Using Diffusers**](./using-diffusers/conditional_image_generation)) or have a look at [**Pipelines**](#pipelines) to get an overview of all supported pipelines and their corresponding papers.
-- Various noise schedulers that can be used interchangeably for the preferred speed vs. quality trade-off in inference. For more information see [**Schedulers**](./api/schedulers/overview).
-- Multiple types of models, such as UNet, can be used as building blocks in an end-to-end diffusion system. See [**Models**](./api/models) for more details 
-- Training examples to show how to train the most popular diffusion model tasks. For more information see [**Training**](./training/overview).
+- State-of-the-art [diffusion pipelines](api/pipelines/overview) that can be run in inference with just a few lines of code.
+- Interchangeable noise [schedulers](api/schedulers/overview) for different diffusion speeds and output quality.
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:26]:
```suggestion
- Interchangeable [ noise schedulers](api/schedulers/overview) for balancing trade-offs between generation speed and quality.
```

File: docs/source/en/index.mdx [hunk 3]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
 
-The following table summarizes all officially supported pipelines, their corresponding paper, and if 
-available a colab notebook to directly try them out.
+## Supported pipelines
 
-| Pipeline | Paper | Tasks | Colab
-|---|---|:---:|:---:|
+| Pipeline | Paper | Tasks |
+|---|---|:---:|
 | [alt_diffusion](./api/pipelines/alt_diffusion) | [**AltDiffusion**](https://arxiv.org/abs/2211.06679) | Image-to-Image Text-Guided Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:54]:
```suggestion
| [alt_diffusion](./api/pipelines/alt_diffusion) | [AltCLIP: Altering the Language Encoder in CLIP for Extended Language Capabilities](https://arxiv.org/abs/2211.06679) | Image-to-Image Text-Guided Generation |
```

I believe providing full names of the papers is more informative and reduces the chances of any confusion.

File: docs/source/en/index.mdx [hunk 4]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
+  </div>
+</div>
 
-The following table summarizes all officially supported pipelines, their corresponding paper, and if 
-available a colab notebook to directly try them out.
+## Supported pipelines
 
-| Pipeline | Paper | Tasks | Colab
-|---|---|:---:|:---:|
+| Pipeline | Paper | Tasks |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:52]:
```suggestion
| Pipeline | Paper / Repository | Tasks |
```

For audio diffusion, we provide a link to the original repository.

File: docs/source/en/index.mdx [hunk 5]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
 | [alt_diffusion](./api/pipelines/alt_diffusion) | [**AltDiffusion**](https://arxiv.org/abs/2211.06679) | Image-to-Image Text-Guided Generation |
-| [audio_diffusion](./api/pipelines/audio_diffusion) | [**Audio Diffusion**](https://github.com/teticio/audio-diffusion.git) | Unconditional Audio Generation | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/teticio/audio-diffusion/blob/master/notebooks/audio_diffusion_pipeline.ipynb)
-| [controlnet](./api/pipelines/stable_diffusion/controlnet) | [**ControlNet with Stable Diffusion**](https://arxiv.org/abs/2302.05543) | Image-to-Image Text-Guided Generation | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/controlnet.ipynb)
+| [audio_diffusion](./api/pipelines/audio_diffusion) | [**Audio Diffusion**](https://github.com/teticio/audio-diffusion.git) | Unconditional Audio Generation |
+| [controlnet](./api/pipelines/stable_diffusion/controlnet) | [**ControlNet with Stable Diffusion**](https://arxiv.org/abs/2302.05543) | Image-to-Image Text-Guided Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:56]:
```suggestion
| [controlnet](./api/pipelines/stable_diffusion/controlnet) | [Adding Conditional Control to Text-to-Image Diffusion Models](https://arxiv.org/abs/2302.05543) | Image-to-Image Text-Guided Generation |
```

File: docs/source/en/index.mdx [hunk 6]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
 | [cycle_diffusion](./api/pipelines/cycle_diffusion) | [**Cycle Diffusion**](https://arxiv.org/abs/2210.05559) | Image-to-Image Text-Guided Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:57]:
```suggestion
| [cycle_diffusion](./api/pipelines/cycle_diffusion) | [Unifying Diffusion Models' Latent Space, with Applications to CycleDiffusion and Guidance](https://arxiv.org/abs/2210.05559) | Image-to-Image Text-Guided Generation |
```

File: docs/source/en/index.mdx [hunk 7]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
 | [dance_diffusion](./api/pipelines/dance_diffusion) | [**Dance Diffusion**](https://github.com/williamberman/diffusers.git) | Unconditional Audio Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:58]: We already have mentioned it above, no? Can be removed, I guess?

File: docs/source/en/index.mdx [hunk 8]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
 | [stable_diffusion_panorama](./api/pipelines/stable_diffusion/panorama) | [**MultiDiffusion**](https://multidiffusion.github.io/) | Text-to-Panorama Generation |
-| [stable_diffusion_pix2pix](./api/pipelines/stable_diffusion/pix2pix) | [**InstructPix2Pix**](https://github.com/timothybrooks/instruct-pix2pix) | Text-Guided Image Editing| 
-| [stable_diffusion_pix2pix_zero](./api/pipelines/stable_diffusion/pix2pix_zero) | [**Zero-shot Image-to-Image Translation**](https://pix2pixzero.github.io/) | Text-Guided Image Editing | 
-| [stable_diffusion_attend_and_excite](./api/pipelines/stable_diffusion/attend_and_excite) | [**Attend and Excite for Stable Diffusion**](https://attendandexcite.github.io/Attend-and-Excite/) | Text-to-Image Generation | 
-| [stable_diffusion_self_attention_guidance](./api/pipelines/stable_diffusion/self_attention_guidance) | [**Self-Attention Guidance**](https://ku-cvlab.github.io/Self-Attention-Guidance) | Text-to-Image Generation | 
+| [stable_diffusion_pix2pix](./api/pipelines/stable_diffusion/pix2pix) | [**InstructPix2Pix**](https://github.com/timothybrooks/instruct-pix2pix) | Text-Guided Image Editing|
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:73]:
```suggestion
| [stable_diffusion_pix2pix](./api/pipelines/stable_diffusion/pix2pix) | [InstructPix2Pix: Learning to Follow Image Editing Instructions](https://arxiv.org/abs/2211.09800) | Text-Guided Image Editing|
```

File: docs/source/en/index.mdx [hunk 9]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
+| [stable_diffusion_pix2pix_zero](./api/pipelines/stable_diffusion/pix2pix_zero) | [**Zero-shot Image-to-Image Translation**](https://pix2pixzero.github.io/) | Text-Guided Image Editing |
+| [stable_diffusion_attend_and_excite](./api/pipelines/stable_diffusion/attend_and_excite) | [**Attend and Excite for Stable Diffusion**](https://attendandexcite.github.io/Attend-and-Excite/) | Text-to-Image Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:75]:
```suggestion
| [stable_diffusion_attend_and_excite](./api/pipelines/stable_diffusion/attend_and_excite) | [Attend-and-Excite: Attention-Based Semantic Guidance for Text-to-Image Diffusion Models](https://arxiv.org/abs/2301.13826) | Text-to-Image Generation |
```

File: docs/source/en/index.mdx [hunk 10]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
+| [stable_diffusion_self_attention_guidance](./api/pipelines/stable_diffusion/self_attention_guidance) | [**Self-Attention Guidance**](https://ku-cvlab.github.io/Self-Attention-Guidance) | Text-to-Image Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:76]:
```suggestion
| [stable_diffusion_self_attention_guidance](./api/pipelines/stable_diffusion/self_attention_guidance) | [Improving Sample Quality of Diffusion Models Using Self-Attention Guidance](https://arxiv.org/abs/2210.00939) | Text-to-Image Generation |
```

File: docs/source/en/index.mdx [hunk 11]
```diff
@@ -16,61 +16,76 @@ specific language governing permissions and limitations under the License.
...
+| [stable_diffusion_safe](./api/pipelines/stable_diffusion_safe) | [**Safe Stable Diffusion**](https://arxiv.org/abs/2211.05105) | Text-Guided Generation |
 | [stable_unclip](./stable_unclip) | **Stable unCLIP** | Text-to-Image Generation |
 | [stable_unclip](./stable_unclip) | **Stable unCLIP** | Image-to-Image Text-Guided Generation |
-| [stochastic_karras_ve](./api/pipelines/stochastic_karras_ve) | [**Elucidating the Design Space of Diffusion-Based Generative Models**](https://arxiv.org/abs/2206.00364) | Unconditional Image Generation | 
+| [stochastic_karras_ve](./api/pipelines/stochastic_karras_ve) | [**Elucidating the Design Space of Diffusion-Based Generative Models**](https://arxiv.org/abs/2206.00364) | Unconditional Image Generation |
 | [unclip](./api/pipelines/unclip) | [Hierarchical Text-Conditional Image Generation with CLIP Latents](https://arxiv.org/abs/2204.06125) | Text-to-Image Generation |
```
Comment (sayakpaul) [inline docs/source/en/index.mdx:87]: UnCLIP comes from KakaoBrain, right? I think we should definitely put a mention here maybe as a footnote.

Cc: @patrickvonplaten

**Review by sayakpaul — APPROVED** [review]
Looks really good!

File: docs/source/en/index.mdx (hunk 11 above)
Comment (stevhliu) [reply docs/source/en/index.mdx:87]: Added a brief note and link to the kakaobrain [unCLIP implementation](https://github.com/kakaobrain/karlo) :)

File: docs/source/en/index.mdx (hunk 3 above)
Comment (stevhliu) [reply docs/source/en/index.mdx:54]: Wow, thanks so much for going through the list and providing the full names!

**Review by patrickvonplaten — APPROVED** [review]
Cool! Feel free to merge :-)

**Pull request merged**